### DISCUSSION TITLE
Create a helm chart repository

### DISCUSSION THREAD

**Discussion by chrootlogin** [openappsec/openappsec#245] [Feature Requests]
Dear OpenAppSec Team

I think the way how helm charts are released should be improved. It would be very useful to have a helm repo which also includes the CRDs so that I can easily update OpenAppSec when there is an update.

Could you please create a helm chart repository?

Best regards.

**Comment by orianelou** [comment]
Hi @chrootlogin, 

Thank you for the suggestion! We agree that this can be improved, and plan to create a Helm Repo by the end of the year, I'll update here on our progress

**Comment by elraro** [comment]
Hello @orianelou,

Will there be a repository before the end of the year?

Best regards

**Comment by orianelou** [comment]
Hi @elraro, 

We plan to release this very soon, in the beginning of 2025

**Reply by anatolek** [reply]
@orianelou I'd like to help and make some changes to helm charts. However, it seems that the current configuration is outdated compared to the one that can be downloaded via

```shell
wget https://downloads.openappsec.io/packages/helm-charts/nginx-ingress/open-appsec-k8s-nginx-ingress-latest.tgz
```

As I understand it, the only option currently available is to create issues for corrections, right?

**Comment by abb-corpay** [comment]
We are now almost halfways through 2025 - can we expect a Helm repository to manifest itself anytime soon? :)

**Comment by orianelou** [comment]
Hi @abb-corpay, @chrootlogin @elraro @anatolek 

This was added as part of our latest 1.1.26 release
